ipvs/netlink.go

```diff
@@ -402,6 +402,14 @@ func (i *Handle) doGetServicesCmd(svc *Service) ([]*Service, error) {
 	return res, nil
 }
 
+// doCmdWithoutAttr a simple wrapper of netlink socket execute command
+func (i *Handle) doCmdWithoutAttr(cmd uint8) error {
```

Comment:
The changes LGTM. But am curious to understand why we should introduce this wrapper when we can add a simple fix in `doCmdwithResponse` 
```
	if s == nil {
            if cmd == ipvsCmdGetDest {
		req.Flags |= syscall.NLM_F_DUMP                    //Flag to dump all messages
                req.AddData(nl.NewRtAttr(ipvsCmdAttrService, nil)) //Add a dummy attribute
          }
     } 
```
Please note the check for `if cmd == ipvsCmdGetDest {` when service == nil.
Will that do the same thing ?

Reply:
Actually, I have tried to reuse `doCmdwithResponse()` before. For example,

```
func (i *Handle) Flush() error {
	// return i.doCmdWithoutAttr(ipvsCmdFlush)
	_, err := i.doCmdwithResponse(nil, nil, ipvsCmdFlush)
	return err
}
```

Then, it will cause an `operation not supported` error. I have not found out why, but I guess 

```
if s == nil {
	req.Flags |= syscall.NLM_F_DUMP                    //Flag to dump all messages
	req.AddData(nl.NewRtAttr(ipvsCmdAttrService, nil)) //Add a dummy attribute
}
```

ipvsCmdFlush is not compatiable with `syscall.NLM_F_DUMP` Flags or `ipvsCmdAttrService` atrribute?

Reply:
@m1093782566 Yes. That could be and hence my suggestion to add the `if cmd == ipvsCmdGetDest {` check before setting the Flag and the Attr.

But again. This is not a bit deal and having such a convenience wrapper as in `doCmdWithoutAttr` is useful as is.